agents/loco_mc_agent.py

```diff
@@ -246,32 +247,46 @@ def step(self):
         super().step()
         self.maybe_dump_memory_to_dashboard()
 
+    def mem_filter(self, mem, filters=None):
```

Comment:
all of these can be done just by modifying the query in lines 262 or 270, no?

Reply:
I guess yes, but we still need a flag signaling that. I did this way for now because I'm unfamiliar with database commands :(